pkgs/applications/video/mkvtoolnix/default.nix

```diff
@@ -25,7 +25,7 @@ stdenv.mkDerivation rec {
     expat
     file xdg_utils boost libebml zlib
     libmatroska libogg libvorbis flac
-    (optional withGUI qt5.qtbase)
+    (optional withGUI qt5.qtbase.dev)
   ];
```

Comment:
@dezgeg Let me see if I understand: `qt5.qtbase`  is a list with more than one element; when it is written explicitly inside the `buildInputs` list it confuses `mkDerivation` and only the first element is used and the other ones are dropped; so it has to be appended to `buildInputs` and then no element is dropped. Is that right?